src/Common/Http/Response.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry @ovr <talk@dmtry.me>
+ */
+declare(strict_types=1);
+
+namespace SocialConnect\Common\Http;
+
+class Response implements ResponseInterface
```

Comment:
I probably missed something but I don't see this class used anywhere.

Reply:
@ADmad Thanks

`ResponseInterface` - was dropped
`Response` - now extends from `GuzzleHttp\Psr7\Response` as `Request`